app/models/app.rb

```diff
@@ -28,7 +28,7 @@ class Build
     def initialize(app_id:, version:, built_at:)
       @app_id = app_id
       @version = AppVersion.new(version)
-      @built_at = built_at
+      @built_at = built_at.is_a?(String) ? Time.zone.parse(built_at) : built_at
```

Comment:
Note that we'll parse anything that's parseable, but clients should definitely choose to represent timestamp in something unambiguous with a time zone.

Reply:
we don't wanna be more prescriptive here and say it's gotta be an iso timestamp?

Reply:
Yeah I dunno, I could, but i don’t have a great precedent for sending 422s back in response to a get. I guess I can still do it and it’ll get the point across to client implementors and fail safer.

Reply:
Note that iso dates don’t have to include minutes and seconds, so maybe I need to do a regex for a canonical form with the precision I want to see.